shell/core/plugin-routes.ts

```diff
@@ -6,20 +6,8 @@ interface RouteInfo {
   route: RouteConfig;
 }
 
-interface RouteInstallInfo {
-  plugin: string;
-  route: RouteConfig;
-}
-
-type RouteInstallHistory = {
-  [route: string]: RouteInstallInfo[]
-}
-
 export class PluginRoutes {
   router: Router;
-  pluginRoutes: RouteConfig[] = [];
-
-  replacedRoutes: RouteInstallHistory = {};
```

Comment:
We don't need to track this information any longer since we don't need to handle uninstall.